### PR TITLE
Image Upgrade on PowerVM

### LINKED ISSUE #144: Update Image Upgrade on PowerVM

### PR DESCRIPTION
**Pull request by AmeyShikerkar**
Fixes: #144
Signed-off-by: Amey Shikerkar <amey.shikerkar@ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/143#" title="Author self-approved">AmeyShikerkar</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **bpradipt** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @bpradipt` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervm).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by AmeyShikerkar** [comment]
Issue : https://github.com/ocp-power-automation/ocp4-upi-powervm/issues/144

File: variables.tf
Comment (bpradipt) [inline variables.tf]: Let's use the same description as in ocp4-playbooks.

File: var.tfvars
Comment (bpradipt) [inline var.tfvars]: Why the proxy variable changes in the PR for image upgrade ?

File: variables.tf
Comment (AmeyShikerkar) [reply variables.tf]: Updated description same as in ocp4-playbooks -  "OCP upgrade image e.g. quay.io/openshift-release-dev/ocp-release-nightly@sha256:xxxxx"

File: var.tfvars
Comment (AmeyShikerkar) [reply var.tfvars]: Code had got merged during rebase command, updated now with latest commit.

**Review by yussufsh — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/143#" title="Author self-approved">AmeyShikerkar</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/143#pullrequestreview-561771945" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervm).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/143#" title="Author self-approved">AmeyShikerkar</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/143#pullrequestreview-562690534" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/143#pullrequestreview-561771945" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervm).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
